### PR TITLE
Arfamomin/jdi 12 healing resources screen

### PR DESCRIPTION
**Pull request by arfamomin**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Just added the two buttons 
Fetched data from Supabase

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
Data is not displayed anywhere 

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @philipye314

### CODE REVIEW

File: src/HealingResources/index.tsx [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
-  return <Text>hello</Text>;
+  const [, setSummaries] = useState<Resource[]>([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      const { data, error } = await supabase
```
Comment (philipye314) [inline src/HealingResources/index.tsx:19]: LOOKS GOOD LOL
